### ISSUE TITLE
Help window enhancements

### ISSUE THREAD

**Issue by akien-mga** [OpenDungeons/OpenDungeons#1098]
Suggestions from bark on the forums: http://forum.freegamedev.net/viewtopic.php?f=36&t=6791#p67642

As a new player that haven't played this type of game before, I'm having a hard time figuring out how to play the game “well”. I notice the help menu which I skimmed through a couple of times before realizing that I have to read it all to hopefully find some of the information I'm looking for.

The help menu should throw what I need in my face, but instead it feels like it's hiding it from me in dense and monotone text that I have to read thoroughly to comprehend. So I'd like to see a cleanup of the in-game help menu. The information in this menu is crucial info to me, but it's scarce (at some points) and hard to read because of the typography.

First off, I suggest rearranging the paragraphs:
1. Basic Workflow
2. Keeper Hand Controls
3. Camera Controls
4. Tips & Tricks
I think this order make more sense, as the Basic Workflow paragraph is more interesting for a new player to read about (how to actually play the game) than information on how to move the camera and such (even though the camera information of course is vital too).

Then I think you should space out the bulletpoints, as they are too close. Give them proper spacing above and below. You could also consider using bright colors on the text of key parts of the bulletpoints (for instance bright yellow). Example:

```
[yellow]Summon workers[/yellow] to do you dirty work...
Use the [yellow]Research Manager (f4)[/yellow] to set the priority...
Once you have a [yellow]workshop[/yellow] set [yellow] traps[/yellow] to protect your...
```

The following sentence is of vital importance to a beginner, but it is hidden at the end of a long bulletpoint far down on the page:

```
”Varied buildings, wealth and great dungeons will attract more powerful creatures”. 
```

This should be a bulletpoint on it's own, and I'd also like to see it elaborated. I'm curious about more information regarding what “great dungeons” are and or how to make them happen.

In the camera section, you use the phrase “Camera translation”. I argue that “camera movement” would make more sense in this instance.

**Comment by hwoarangmy** [comment]
Indeed, I guess it is not very clear to new players how to play be reading only this help page. However, I guess the proper way would be to script a level (ie creating a campaign). That will be something we will have to discuss.

**Comment by Bertram25** [comment]
I would also maybe add tabs to the help window, so we can split subjects more clearly?

**Comment by hwoarangmy** [comment]
Yes. As long as the tab labels are explicit, it would make it more readable.
